tests/unit_test/app/routes/test_git_tokens.py

```
@@ -0,0 +1,342 @@
+				call_args = mock_git_label.create.call_args
+				assert call_args.kwargs["user_id"] == mock_authenticated_user.id
+
+
+class TestDeleteGitLabelEndpoint:
+	"""Test cases for DELETE /api/v1/git_tokens/{git_label_id} endpoint."""
+	
+	@staticmethod
+	def get_url(git_label_id: str) -> str:
+		return f"/api/v1/git_tokens/{git_label_id}"
+	
+	def test_delete_git_label_success(self, client, sample_token_id, mock_authenticated_user):
```

Pylint (3.3.7)

[warning] Bad indentation. Found 1 spaces, expected 4

(W0311)